Example/DownloadToGo/TracksViewController.swift

```diff
@@ -41,7 +41,7 @@ class TracksViewController: UITableViewController {
         self.delegate?.trackViewControllerDidSelectAudioTrack(self.selectedAudioTrack, andTextTrack: self.selectedTextTrack)
     }
     
-    func dismissTracksVC() {
+    @objc func dismissTracksVC() {
```

Comment:
Why do we need `@objc` on these functions? It's in the app, no objective-c code will call them.

Reply:
Because we are calling this function from inside #selector()